test/parallel/test-child-process-exec-std-encoding.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const cp = require('child_process');
+const stdoutData = 'foo';
+const stderrData = 'bar';
+const expectedStdout = `${stdoutData}\n`;
+const expectedStderr = `${stderrData}\n`;
+
+if (process.argv[2] === 'child') {
+  // The following console calls are part of the test.
+  console.log(stdoutData);
```

Comment:
What do you think about using `process.{stdout,stderr}.write()`?

Reply:
process.std.write is ok. But in other test-child-process tests use the console, like [encoding](https://github.com/nodejs/node/blob/master/test/parallel/test-child-process-exec-encoding.js#L12),[env](https://github.com/nodejs/node/blob/master/test/parallel/test-child-process-exec-env.js#L34).